executor/distsql.go

```diff
@@ -334,7 +336,7 @@ type IndexLookUpExecutor struct {
 	memTracker *memory.Tracker
 
 	// isCheckOp is used to determine whether we need to check the consistency of the index data.
-	isCheckOp bool
+	checkIndexValue
```

Comment:
How about use `*checkIndexValue` and remove `checkIndexValue` filed `isCheckOp`, then use `w.checkIndexValue != nil` to check.

BYW, need to update the comment in line#339, 622, 738.

Reply:
`checkIndexValue.genExprs` is assigned in a shared function, so we can't use `w.checkIndexValue != nil` to check if we need to check the index value. So I don't update it.

Reply:
Maybe use pointer is better? At most cases, this checkIndexValue field is not necessary.